src/app/components/ItemPage/ItemPage.jsx

```diff
@@ -67,145 +67,66 @@ class ItemPage extends React.Component {
     });
   }
 
-  getDisplayFields(record, data) {
-    if (!data.length) return [];
+  render() {
+    const createAPIQuery = basicQuery(this.props);
+    const record = this.props.bib ? this.props.bib : this.props.item;
+    const title = record.title && record.title.length ? record.title[0] : '';
+    const holdings = LibraryItem.getItems(record);
 
-    const detailFields = [];
-    data.forEach((f) => {
+    const displayFields = [
+      { label: 'Title', field: 'title' },
+      { label: 'Title (display)', field: 'titleDisplay' },
```

Comment:
There's also `titleAlt`, `suppressed`, `idHathi `, `hathiVols`. Can you add those too?